### PR TITLE
[SPARK-33796][DOCS] Show hidden text from the left menu of Spark Doc

### PR DESCRIPTION
**Pull request by gengliangwang**
### What changes were proposed in this pull request?

If the text in the left menu of Spark is too long, it will be hidden.
sql1

This PR is to fix the style issue.

### Why are the changes needed?

Improve the UI of Spark documentation.

### Does this PR introduce _any_ user-facing change?

No
### How was this patch tested?

Manual test
After changes:
sql2

### CODE REVIEW

File: docs/css/main.css [hunk 1]
```diff
@@ -333,10 +333,6 @@ a.anchorjs-link:hover { text-decoration: none; }
   overflow-y: scroll;
 }
 
-.left-menu {
-  width: 399px;
```
Comment (gengliangwang) [inline docs/css/main.css:337]: The width is specified as 210px in its parent `.left-menu-wrapper`

**Review by HyukjinKwon — APPROVED** [review]

**Comment by HyukjinKwon** [comment]
Thanks @gengliangwang for fixing this!

**Comment by HyukjinKwon** [comment]
Merged to master, branch-3.1 and branch-3.0.

**Comment by HyukjinKwon** [comment]
@gengliangwang, should we fix it in branch-3.0 too? it has a conflict.

**Comment by gengliangwang** [comment]
@HyukjinKwon sure, I will do it.

**Pull request closed without merging**